### ISSUE TITLE
Assistant alt-titles gone, Job Extra Accesses gone.

### ISSUE THREAD

**Issue by Natje0** [VOREStation/VOREStation#2321]
#### Brief description of the issue
Assistant titles are gone.
Jobs no longer have extra accesses within their departments (Robotics > R&D, Botany > Kitchen)
It seems they've been set to need "All" accesses listed on them, instead of "One". Or something like that.

#### What you expected to happen
Botanists to be able to serve as a ghetto-chef

#### What actually happened
Starvation.

#### Steps to reproduce
- (Step 1) Spawn as Botanist/Roboticist/probably a lot of others too
- (Step 2) Be unable to enter Kitchen/R&D
- (Step 3) Cry.

#### Code Revision
- (Found using the "Show Server Revision" verb under the OOC tab.)
 506e28bdf3a78e916863360f987f394efc9d65fb 
#### Anything else you may wish to add:
- (Location if it's a mapping issue, screenshots, sprites, etc.)

**Comment by Cameron653** [comment]
Kitchen airlock: https://github.com/VOREStation/VOREStation/blob/e460250a1dc2464b32c9b8ba4da1161c7418edb6/code/game/machinery/doors/airlock.dm#L216-L220

Botanist accesses:
https://github.com/VOREStation/VOREStation/blob/master/code/game/jobs/job/civilian.dm#L48-L49
```c
	access = list(access_hydroponics, access_bar, access_kitchen)
	minimal_access = list(access_hydroponics)
```

Where access giving is handled (deciding between minimal and access)
https://github.com/VOREStation/VOREStation/blob/163c686d183ca817998f34b6d9362cc29b93f7a8/code/game/jobs/job/job.dm#L88-L92

TL;DR: Config file might have been changed. I don't see any reason botanists wouldn't have kitchen access otherwise.


Onto assistants:
https://github.com/VOREStation/VOREStation/blob/master/code/game/jobs/job/assistant.dm#L15-L35
Something got horribly fucked here when it was merging Polaris and our file together. Going to fix it.

**Comment by Screemonster** [comment]
Engineers can't get into the workshop either, and atmotechs can't open electrical lockers.
